### ISSUE TITLE
Cannot re-create a deleted repository.

### ISSUE THREAD

**Issue by sometowngeek** [github/VisualStudio#1018]
- GitHub Extension for Visual Studio  version: 2.2.0.11
- Visual Studio version: 12.2.26430.14

__What happened__ (with steps, logs and screenshots, if possible)
I tried deleting an existing repo from GitHub, then create a new repo, using the same name "SOExamples"; however, GitHub Extension refuses to allow me to use the same name.

1. Delete an existing repo via github.com
2. Delete the .git folder from the solution folder
3. Open Visual Studio 2017
4. From Team Explorer >> Manage Connections, click on "Create" underneath the Github header
5. Type in the same name of the deleted repo.
    * GitHub Extension refuses to create the repo.
6. Type in a different name
    * GitHub Extension says it can create the repo.

What did I do wrong? What can I do to fix this?

# Snapshots:
![vs-githuberror](https://user-images.githubusercontent.com/11671755/27498771-364ba652-5826-11e7-83f2-8eaaa878c55c.png)
## GitHub Extension Refuses
![vs-githuberror-2](https://user-images.githubusercontent.com/11671755/27498857-9ed4e9ae-5826-11e7-8706-309d81051d1e.png)
## GitHub Extension Accepts
![vs-githuberror-3](https://user-images.githubusercontent.com/11671755/27498905-d0e9a920-5826-11e7-99cb-537ead01173d.png)

**Comment by grokys** [comment]
Hi @sometowngeek,

I've just tried this and I'm unable to reproduce this problem locally. I suspect it may be caused by caching, though that doesn't explain why I can't reproduce it. 

Could you try closing Visual Studio and deleting the following files/folders and then trying again:

`%LocalAppData%\GitHubVisualStudio\BlobCache`
`%AppData%\GitHubVisualStudio\BlobCache`
`%AppData%\GitHubVisualStudio\api.github.com.cache.db`

**Comment by sometowngeek** [comment]
That fixed the problem! Thank you! :)

**Comment by paladique** [comment]
Glad that worked! Closing this out!

**Comment by sometowngeek** [comment]
I think this might actually be an issue with an existing directory...

For instance, if I already have a project in C:\SomePath\ProjectName, then I try to create a GitHub repository in the same path, it won't let me.

Is this supposed to happen or should I be able to create a repo, given the .git folder doesn't exist yet?

**Comment by oslotboom** [comment]
I am still having this problem, and deleting the cache files did not correct it.
What I did is
1. Created a repository on Git via Visual Studio
2. The Git repository did not bind to my local code and I could not solve the issue, so I deleted the repository on the Github site to start over.
3. I can no longer create a new repository via Visual Studio with the same name as the deleted repository. I deleted the cache files, logged out of Github, restarted Visual Studio and rebooted, but entering the original repository name still gives the error icon and disables the Create button
